src/EditorFeatures/Core.Wpf/InlineDiagnostics/AbstractDiagnosticsTaggerProvider.SingleDiagnosticKindPullTaggerProvider.cs

```diff
@@ -91,9 +91,6 @@ protected sealed override ITaggerEventSource CreateEventSource(ITextView? textVi
         private async Task ProduceTagsAsync(
             TaggerContext<TTag> context, DocumentSnapshotSpan documentSpanToTag, CancellationToken cancellationToken)
         {
-            if (!_callback.IsEnabled)
```

Comment:
no concept of 'enabled' or not.  inline diagnostics aren't enabled/disabled depending on if we're using lsp diagnostics or not.